src/components/Breadcrumbs/__stories__/Variants.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import { Story } from '@storybook/react'
+import { ComponentProps } from 'react'
+import Breadcrumbs, { breadcrumbsVariants } from '..'
+
+export const Variants: Story<ComponentProps<typeof Breadcrumbs>> = props => (
+  <>
+    {breadcrumbsVariants.map(variant => (
+      <>
+        <b>{variant}:</b>
+        <Breadcrumbs {...props} key={variant} selected={1} variant={variant}>
+          <Breadcrumbs.Item>Step 1</Breadcrumbs.Item>
+          <Breadcrumbs.Item>Step 2</Breadcrumbs.Item>
+          <Breadcrumbs.Item>Step 3</Breadcrumbs.Item>
+        </Breadcrumbs>
+        <br />
+      </>
+    ))}
+  </>
+)
```

Comment:
I suggest you add a decorator, so you can remove fragments and `br` elements, it will produce a much cleaner code snippet:

```tsx
export const Variants: Story<ComponentProps<typeof Breadcrumbs>> = props => breadcrumbsVariants.map(variant => (
        <Breadcrumbs {...props} key={variant} selected={1} variant={variant}>
          <Breadcrumbs.Item>Step 1</Breadcrumbs.Item>
          <Breadcrumbs.Item>Step 2</Breadcrumbs.Item>
          <Breadcrumbs.Item>Step 3</Breadcrumbs.Item>
        </Breadcrumbs>
    ))
)

Types.decorators = [
  StoryComponent => (
    <Stack gap={2}>
      <StoryComponent />
    </Stack>
  ),
] as DecoratorFunction<JSX.Element>[]
```

See `Alert/stories/Variants.stories` for a reference :)

Reply:
Thanks, updated!